src/components/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import SearchBar from './SearchBar';
+
+describe('SearchBar', () => {
+  beforeEach(() => {
+    render(<SearchBar />);
+  });
+
+  it('renders an input element', () => {
+    const inputElement = screen.getByPlaceholderText(/search/i);
+    expect(inputElement).toBeInTheDocument();
+  });
+
+  it('changes text when typed into', () => {
+    const typedText = 'abcde';
+
+    const inputElement = screen.getByPlaceholderText(/search/i);
+    fireEvent.change(inputElement, { target: { value: typedText } });
+
+    expect(inputElement.value).toBe(typedText);
+  });
```

Comment:
you did a really good use of testing library here, but maybe right now there is not too much to test because there is no functionality, and this tested functionality is the default from HTML, so it is not necessary to be tested